README.md

```diff
@@ -134,7 +141,7 @@ let _ = App.empty
 Compile with:
 
 ```
-$ corebuild -pkg opium middleware_ua.native
+$ ocamlbuild -pkg opium.unix middleware_ua.native
```

Comment:
I think ocamlfind support is enabled by default in a modern ocamlbuild; this does work at least on my computer.